installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{- define "service.image" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- $registryName := "" -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $tag := include "service.tag" (dict "imageRoot" .imageRoot "global" .global "defaultTag" .defaultTag) -}}
+{{- if .global }}
+    {{- if .global.registry }}
+     {{- $registryName = .global.registry -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.registry -}}
+  {{- $registryName = .imageRoot.registry -}}
+{{- end -}}
+{{- if $registryName }}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- else -}}
+{{- printf "%s:%s" $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*Return the proper service image tag*/}}
+{{- define "service.tag" -}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.lighthouseService.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
```

Comment:
should this be `service.tag` instead of `common.images.image` in the comment?

Reply:
yes you are right thanks and same for service.tag